gradle/buildcontainer.gradle

```diff
@@ -0,0 +1,44 @@
+// Deprecated. To be removed
```

Comment:
agreed, we definitely don't want gradle doing the container builds.

Reply:
Yes these tasks will not be executed by default in the build process. Only if explicitly use the **run** profile while running a gradle build using -Dorg.gradle.project.profile=run